docs/schemaform/form-config.md

```diff
@@ -180,22 +180,29 @@ We've also been adding some additional uiSchema functionality not found in the r
   'ui:required': function (formData) {
     return true || false;
   },
-  
+
   // This is an array of validation functions or objects that can be used to add validation
   // that is not possible through JSON Schema. See below for the properties passed
   // to the validation functions and how to use them.
   'ui:validations': [
-    function (errors, fieldData, pageData, fieldSchema, errorMessages) {
+    /**
+     * Note the difference between the three data parameters:
+     *
+     * @param {any} fieldData The data for the current field being validated
+     * @param {object} pageData The data for all the fields on the current page
+     * @param {object} formData The data for all the fields in every page
+     */
+    function (errors, fieldData, pageData, formData fieldSchema, errorMessages) {
```

Comment:
OCD!
MUST REMOVE ALL TRAILING WHITESPACE!

The `formData` is basically all the pages' `data` contents slapped together in one object. That's all the more it's flattened. `pageData` is just the `data` on the current page, so they both have essentially the same structure. `formData` just has more`.

....bloody commas....should lint these .md files. >_>